Fixtures/products.json

```diff
@@ -0,0 +1,20 @@
+{
+  "products": [{
+    "name": "Flap Bag",
+    "designer": "Chanel",
+    "product_price": "$5,000",
+    "site_url": "https.chanel.com/flap-bag",
+    "condition": "New",
+    "material": "Leather, metal",
+    "size": "OS",
+    "dimensions": "10 x 12 in",
+    "sku": "TestSKU-1",
+    "season": "SS 2020",
+    "collection": "Seasonal",
+    "category": "bags",
+    "brand": "Chanel",
+    "images": ["https://via.placeholder.com/150", "https://via.placeholder.com/150"],
+    "stock": [{"color": "black", "quantity": 3}, {"color": "white", "quantity": 5}, {"color": "burgandy", "quantity": 4}, {"color": "black & white", "quantity": 0}, {"color": "black, white, & gray", "quantity": 4}],
+    "keywords": ["Flap Bag", "Chanel", "Leather", "metal", "TestSKU-1", "SS2020", "Seasonal"]
+    }]
```

Comment:
Will add more and varied data. Is this also the right way to go about mocking/stubbing the data?

Reply:
@jmyers0022 Since we are starting with the UI in this case we'll use the results of this process to define what needs to come back from the api endpoint and subsequently used as the test response for the django app.